File: backend/voting-app/src/main/java/com/tu/votingapp/dto/response/elections/CandidateResponseDTO.java
package com.tu.votingapp.dto.response.elections;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CandidateResponseDTO {
    private Long id;
    private String name;
    private String bio;
    private Long electionId;
    private int votesCount;
    private String imageUri;
    private String position;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/elections/CandidateResponseDTO.kt
package com.example.android.dto.response.elections

import kotlinx.serialization.Serializable

@Serializable
data class CandidateResponseDTO(
    val id: Long,
    val name: String,
    val bio: String?,
    val electionId: Long,
    val votesCount: Int,
    val imageUri: String?,
    val position: String?
)